src/domain/backbone/backbone.validator.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { faker } from '@faker-js/faker';
+import { HttpException } from '@nestjs/common';
+import backboneFactory from '../balances/entities/__tests__/backbone.factory';
+import { JsonSchemaService } from '../schema/json-schema.service';
+import { ValidationErrorFactory } from '../schema/validation-error-factory';
+import { BackboneValidator } from './backbone.validator';
+
+const mockValidationErrorFactory = jest.mocked({
+  from: jest.fn(),
+} as unknown as ValidationErrorFactory);
+
+const validationFunction = jest.fn();
+const mockJsonSchemaService = jest.mocked({
+  addSchema: jest.fn(),
+  compile: jest.fn().mockImplementation(() => validationFunction),
+} as unknown as JsonSchemaService);
+
+describe('Backbone validator', () => {
+  const validator = new BackboneValidator(
```

Comment:
I agree in some cases this could be useful, but I think doing this on a regular basis is not necessary and could hurt performance while running tests (for N test cases, the class under test will be built N times). I assume the classes under test (services, repositories, validators, whatever) should be stateless in most cases. I'd even say having internal states in that classes could be a code smell.